.github/workflows/release.yaml

```diff
@@ -112,7 +112,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Replace Variables
+      - name: Replace Variables # why do we need this?
```

Comment:
Because the compose spec is like https://github.com/omnistrate/ci-cd-example/blob/236bef9a1fc5c98cdcdc91115c2dda23e8e78cc2/compose.free.yaml#L12 . We need to replace the placeholders to make it work. Or we can consider change the method here to not use placeholders.